Source/WebKit/UIProcess/API/Cocoa/_WKAutomationSessionConfiguration.h

```diff
@@ -35,6 +35,7 @@ WK_CLASS_AVAILABLE(macos(10.13.4), ios(12.2))
 @property (nonatomic) BOOL acceptInsecureCertificates;
 @property (nonatomic) BOOL allowsInsecureMediaCapture;
 @property (nonatomic) BOOL suppressesICECandidateFiltering;
+@property (nonatomic) BOOL alwaysAllowAutoplay;
```

Comment:
I do wonder if this is an oversimplification, whether we should expose all three options here…

I could imagine a web developer wanting to actually set the autoplay policy to "deny" to test that playback can be started.

Not sure I have particularly strong opinions here, but maybe worth considering…

Reply:
I tend to agree that multiple behaviors could be desired here.

In the past month AutoplayPolicy.h was added:
```
enum class AutoplayPolicy : uint8_t {
    Default, // Uses policies specified in document settings.
    Allow,
    AllowWithoutSound,
    Deny,
};
```

So I think we can go down the route of making an "overrideAutoplayPolicy" with "deny", "allow", "allow-muted". Or, we can create several boolean capabilities---one for each override value---which are mutually exclusive.